src/components/projects/wbs-details/work-package-container/work-package-buttons/work-package-buttons.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { useContext } from 'react';
+import { Button } from 'react-bootstrap';
+import { EditModeContext, EditModeProps } from '../work-package-container';
+import styles from './work-package-buttons.module.css';
+
+const WorkPackageButtons: React.FC<EditModeProps> = (props) => {
+  const editMode = useContext(EditModeContext);
+  return (
+    <div className={`mx-4 my-3 ${styles.workPackageActionButtonsContainer}`}>
+      <Button className={styles.workPackageActionButton}>New Change Request</Button>
```

Comment:
let's either comment this button out, or add functionality to make it redirect to the new change request page

Reply:
the new change request page isn't on this branch - I'll comment it out for now.